add program chair console

mspector opened the pull request:
this is mostly just a copy of the UAI program chair console.

mspector commented:
@zbialecki @melisabok I posted this webfield in the dev instance, you can check it out by logging in as programchair@openreview.net (password=1234)

melisabok commented:
Cool, it looks nice!

melisabok commented:
How much did you have to change from the UAI file? May be is another thing to move to a common place or create a template(located in the super script?) to be reused in future conferences instead of copy/paste.

mspector commented:
I didn't have to change much, just variable names. I think it could be made generic.

mspector commented:
I made a few tweaks to make this more easily generalizable, mostly just moving constants, but I also renamed the table class from "uai-console-table" to "console-table". I modified the less file that formats it in this PR: https://github.com/iesl/openreview/pull/798

melisabok commented on venues/ICLR.cc/2018/Conference/webfield/programchairWebfield.js:
```diff
@@ -0,0 +1,815 @@
+  var userId = $(this).data('userId');
+  var forumUrl = $(this).data('forumUrl');
+  var postData = {
+    subject: 'ICLR 2018 Reminder',
```
we could get this 'ICLR 2018' from a variable

melisabok commented on venues/ICLR.cc/2018/Conference/webfield/programchairWebfield.js:
```diff
@@ -0,0 +1,815 @@
+  var forumUrl = $(this).data('forumUrl');
+  var postData = {
+    subject: 'ICLR 2018 Reminder',
+    message: 'This is a reminder to please submit your official reviews for ICLR 2018. ' +
```
same here

melisabok commented on venues/ICLR.cc/2018/Conference/webfield/programchairWebfield.js:
```diff
@@ -0,0 +1,815 @@
+// CONFERENCE + '/-/Paper' + number + '/Official_Review'
+
+// Constants
+var HEADER_TEXT = 'ICLR 2018 Program Chairs Console';
```
Constants are the ICLR related things. It would be nice to save this in a template, what do you think?.

mspector commented:
@melisabok 

I'm working on changes to admin/superuser-init.py to make fit this file (and others) into a template form, and I'm finding that there are a lot of changes that I want to make. I wonder if we should merge this, and I can make a separate PR with the refactored version of superuser-init ?

Otherwise, I'll close this PR so that I can keep working on it.
